engine/src/main/java/org/hibernate/search/spi/SearchIntegratorBuilder.java

```diff
@@ -499,37 +497,6 @@ private static IndexingMode defineIndexingMode(SearchConfiguration cfg) {
 		return IndexingMode.fromExternalRepresentation( indexingStrategy );
 	}
 
-	private ErrorHandler createErrorHandler(SearchConfiguration searchConfiguration) {
-		Object configuredErrorHandler = searchConfiguration.getProperties().get( Environment.ERROR_HANDLER );
```

Comment:
Ok so you want to wrap whatever is being configured by the user. I had missed that.
[Services won't fit this either then - although a Service is 1 per integrator, unless you want to provide the same instance to each]

Do you need to handle some error internally in Infinispan, while still forward to the user other errors?